### ISSUE TITLE
l10n out of sync on transifex

### ISSUE THREAD

**Issue by wengxt** [owncloud/music#665]
I mean https://www.transifex.com/owncloud-org/owncloud/dashboard/. I noticed some l10n bug in zh_CN but I didn't find the corresponding entry on transifex. Not sure if there's another place for host translation.

**Comment by paulijar** [comment]
Yes, this is a known issue which is being tracked at https://central.owncloud.org/t/owncloud-music-app-translations/14881.

**Comment by Scubage** [comment]
Are there any news, about it? Would like to translate german.

**Comment by paulijar** [comment]
No news, but I posted another question on the ownCloud forum. So let's see...
